homeassistant/components/schlage/select.py

```diff
@@ -0,0 +1,86 @@
+"""Platform for Schlage select integration."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import SchlageDataUpdateCoordinator
+from .entity import SchlageEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class SchlageSelectEntityDescription(SelectEntityDescription):
+    """Entity description for a Schlage select."""
+
+
+_SECONDS_TO_OPTIONS = {
```

Comment:
The string mapping should be in strings.json so it's translatable. motionblinds_ble has a good example: https://github.com/dknowles2/ha-core/blob/dev/homeassistant/components/motionblinds_ble/strings.json#L61-L70

Reply:
Thanks!